exercises/grade-school/README.md

```diff
@@ -22,6 +21,18 @@ In the end, you should be able to:
 Note that all our students only have one name.  (It's a small town, what
 do you want?)
 
+## For bonus points
+
+Did you get the tests passing and the code clean? If you want to, these
+are some additional things you could try:
+
+- If you're working in a language with mutable data structures and your
```

Comment:
I don't think this applies to this exercise - given the interface, the implementation more or less has to be immutable. I suggest not adding any of the new text under "For bonus points"

Reply:
I'll fork the exercise so that this track doesn't get the changes.